index.html

```diff
@@ -4,9 +4,61 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles/index.css" rel="stylesheet" />
     <title>Weather Report</title>
 </head>
 <body>
-    
-</body>
-</html>
\ No newline at end of file
+    <body>
+        <header>
+            <div class="text">
+                <h1>Weather Report </h1>
+                <h2 id="cityQuote"> Weather report for the city of: <span id="display-name"> ... </span> </h2>
+            </div>
+        </header>
+        <section class="top-box">
+            <div class="grid-item-1">
+                <label for="weathergarden">WEATHER GARDEN</label>
+                <p id="display-sky">🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧 </p>
+                <section id="landscape-container">🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃</section>
+            </div>
+        </section>
+        <section class="boxes">
+            <div class="grid-container">
+                <div class="grid-item">
+                    <label for="cityid">CITY</label>
```

Comment:
I suspect you meant to have this be a `h3` and not a `label` since it's pretty far from the widget it labels.